projects/portal/src/app/models/faucets/faucet.model.ts

```diff
@@ -10,6 +10,6 @@ export type FaucetResponse = {
   transfers: {
     coin: string;
     status: string;
-    message?: string;
```

Comment:
@YasunoriMATSUOKA 
Is it a deliberate fix?

Reply:
@mkXultra 
Yes. I debug this and confirm the key was wrong. So it should be fixed.

Response example

```json
{"transfers":[{"coin":"100ubtc","status":"error","error":"Error: RPC error -32603 - Internal error: timed out waiting for tx to be included in a block\nUsage:\n  ununifid tx bank send [from_key_or_address] [to_address] [amount] [flags]\n\nFlags:\n  -a, --account-number uint      The account number of the signing account (offline mode only)\n  -b, --broadcast-mode string    Transaction broadcasting mode (sync|async|block) (default \"sync\")\n      --dry-run                  ignore the --gas flag and perform a simulation of a transaction, but don't broadcast it\n      --fee-account string       Fee account pays fees for the transaction instead of deducting from the signer\n      --fees string              Fees to pay along with transaction; eg: 10uatom\n      --from string              Name or address of private key with which to sign\n      --gas string               gas limit to set per-transaction; set to \"auto\" to calculate sufficient gas automatically (default 200000)\n      --gas-adjustment float     adjustment factor to be multiplied against the estimate returned by the tx simulation; if the gas limit is set manually this flag is ignored  (default 1)\n      --gas-prices string        Gas prices in decimal format to determine the transaction fee (e.g. 0.1uatom)\n      --generate-only            Build an unsigned transaction and write it to STDOUT (when enabled, the local Keybase is not accessible)\n  -h, --help                     help for send\n      --keyring-backend string   Select keyring's backend (os|file|kwallet|pass|test|memory) (default \"test\")\n      --keyring-dir string       The client Keyring directory; if omitted, the default 'home' directory will be used\n      --ledger                   Use a connected Ledger device\n      --node string              \u003chost\u003e:\u003cport\u003e to tendermint rpc interface for this chain (default \"tcp://localhost:26657\")\n      --note string              Note to add a description to the transaction (previously --memo)\n      --offline                  Offline mode (does not allow any online functionality\n  -o, --output string            Output format (text|json) (default \"json\")\n  -s, --sequence uint            The sequence number of the signing account (offline mode only)\n      --sign-mode string         Choose sign mode (direct|amino-json), this is an advanced feature\n      --timeout-height uint      Set a block timeout height to prevent the tx from being committed past a certain height\n  -y, --yes                      Skip tx broadcasting prompt confirmation\n\nGlobal Flags:\n      --chain-id string     The network chain ID\n      --home string         directory for config and data (default \"/root/.ununifi\")\n      --log_format string   The logging format (json|plain) (default \"plain\")\n      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default \"info\")\n      --trace               print out full stack trace on errors\n\n: exit status 1"}]}
```

Reply:
I got it.Thank you